DESIGN.md

```diff
@@ -341,7 +341,7 @@ binary
 ~~~
 
 json
-~~~ json
```

Comment:
I'd prefer to leave the syntax highlighting marker there

Reply:
The marker indicates that the block following is valid json, which it isn't. All the red blocks in the rendered view are error highlighters, it's confusing when using a highlighting editor, because as long as the invalid blocks are in the source, the editor will always report "there are problems in this file".